.azure-pipelines/all.yml

```diff
@@ -132,10 +132,10 @@ jobs:
       displayName: Puma
       os: linux
     - checkName: purefa
-      displayName: purefa
```

Comment:
If you update your branch with master, and remove these changes, validation should pass.